internal/fluentbit/config/builder/config_builder.go

```diff
@@ -38,6 +38,7 @@ func BuildFluentBitConfig(pipeline *telemetryv1alpha1.LogPipeline, config Builde
 
 	var sb strings.Builder
 	sb.WriteString(createInputSection(pipeline, includePath, excludePath))
+	sb.WriteString(createCustomMultilineFilters(pipeline))
 	sb.WriteString(createRecordModifierFilter(pipeline))
 	sb.WriteString(createKubernetesFilter(pipeline))
 	sb.WriteString(createCustomFilters(pipeline))
```

Comment:
Let's rename the function to make it more explicit that we are not dealing with multiline parsers here
```suggestion
	sb.WriteString(createCustomFiltersExceptMultiline(pipeline))
```